docs/guides/sdks/android-sdk/privacy-preserving-android.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Preserve user privacy"
+slug: "preserve-user-privacy"
+category: 5f9705393c689a065c409b23
+parentDoc: 5fa0443749be540011850e51
+excerpt: "Learn how to preserve user privacy in the Android SDK."
+hidden: false
+createdAt: "2024-01-07T19:00:15.000Z"
+updatedAt: "2024-01-07T19:00:15.000Z"
+order: 10
+---
+
+# Preserve user privacy
+
+For a general introduction to privacy-preserving methods in the AppsFlyer SDK, see [Preserving user privacy](https://dash.readme.com/project/hc/v0.1/docs/preserve-user-privacy-1) (under Getting started).
```

Comment:
This cannot be the correct link. This is the link to the page in the dashboard, not the published page